composer.json

```diff
@@ -15,7 +15,8 @@
         "ezsystems/comments-bundle": "@dev",
         "ezsystems/platform-ui-assets-bundle": "~0.1",
         "whiteoctober/breadcrumbs-bundle": "~1.0.1",
-        "knplabs/knp-menu-bundle": "1.*"
+        "knplabs/knp-menu-bundle": "1.*",
+        "components/bootstrap": "3.3.2"
```

Comment:
Should be set to "~3.3.2" so we get patch updates without having to change anything.